### ISSUE TITLE
"Field 'browser' doesn't contain a valid alias configuration"

### ISSUE THREAD

**Issue by mmmavis** [Pomax/react-formbuilder#18]
"Field 'browser' doesn't contain a valid alias configuration" error when importing `react-formbuilder` to a project (in this case, Pulse). Log below:

```bash
mavisou:~/Development/network-pulse[issue-309-form*]$ npm start

> network-pulse@2.0.0 start /Users/mavisou/Development/network-pulse
> npm run build && run-p server watch:**


> network-pulse@2.0.0 build /Users/mavisou/Development/network-pulse
> run-s bootstrap build:**


> network-pulse@2.0.0 bootstrap /Users/mavisou/Development/network-pulse
> shx rm -rf dist && shx mkdir dist && shx rm -f config/env.generated.json


> network-pulse@2.0.0 build:config /Users/mavisou/Development/network-pulse
> node js/build-env-config.js > config/env.generated.json


> network-pulse@2.0.0 build:client /Users/mavisou/Development/network-pulse
> run-s build-client:**


> network-pulse@2.0.0 build-client:js /Users/mavisou/Development/network-pulse
> webpack --config webpack.config.js --optimize-minimize --display-error-details

Hash: 806a79e0f77e0f475005
Version: webpack 2.2.1
Time: 9517ms
    Asset    Size  Chunks                    Chunk Names
bundle.js  522 kB       0  [emitted]  [big]  main
   [4] ./~/react/react.js 56 bytes {0} [built]
  [15] ./~/react-router/es/index.js 1.46 kB {0} [built]
  [19] ./~/react-router/es/RouteUtils.js 2.43 kB {0} [built]
  [27] ./~/react-router/es/PatternUtils.js 7.34 kB {0} [built]
  [29] ./~/react/lib/React.js 2.69 kB {0} [built]
  [37] ./~/react-ga/src/index.js 12.8 kB {0} [built]
 [221] ./~/react-ga/src/utils/console/warn.js 78 bytes {0} [built]
 [222] ./~/react-ga/src/utils/trim.js 203 bytes {0} [built]
 [225] ./~/react-router/es/Redirect.js 2.6 kB {0} [built]
 [227] ./~/react-router/es/createMemoryHistory.js 637 bytes {0} [built]
 [230] ./~/react-router/es/useRouterHistory.js 282 bytes {0} [built]
 [234] ./routes.jsx 5.26 kB {0} [built]
 [235] ./~/react-dom/index.js 59 bytes {0} [built]
 [352] ./~/react-router/es/IndexRoute.js 1.38 kB {0} [built]
 [382] ./main.jsx 1.38 kB {0} [built]
    + 368 hidden modules

ERROR in ./pages/add/add.jsx
Module not found: Error: Can't resolve 'react-formbuilder' in '/Users/mavisou/Development/network-pulse/pages/add'
resolve 'react-formbuilder' in '/Users/mavisou/Development/network-pulse/pages/add'
  Parsed request is a module
  using description file: /Users/mavisou/Development/network-pulse/package.json (relative path: ./pages/add)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/mavisou/Development/network-pulse/package.json (relative path: ./pages/add)
    resolve as module
      /Users/mavisou/Development/node_modules doesn't exist or is not a directory
      /Users/mavisou/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      /Users/mavisou/Development/network-pulse/pages/node_modules doesn't exist or is not a directory
      /Users/mavisou/Development/network-pulse/pages/add/node_modules doesn't exist or is not a directory
      looking for modules in /Users/mavisou/Development/network-pulse/node_modules
        using description file: /Users/mavisou/Development/network-pulse/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/mavisou/Development/network-pulse/package.json (relative path: ./node_modules)
          using description file: /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/package.json (relative path: .)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder is not a file
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder.json doesn't exist
            as directory
              existing directory
                using path: /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index
                  using description file: /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/package.json (relative path: ./index)
                    no extension
                      Field 'browser' doesn't contain a valid alias configuration
                      /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index doesn't exist
                    .js
                      Field 'browser' doesn't contain a valid alias configuration
                      /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index.js doesn't exist
                    .json
                      Field 'browser' doesn't contain a valid alias configuration
                      /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index.json doesn't exist
                use ./dist/react-formbuilder.js from main in package.json
                  using description file: /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/package.json (relative path: .)
                    Field 'browser' doesn't contain a valid alias configuration
                  after using description file: /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/package.json (relative path: .)
                    using description file: /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/package.json (relative path: ./dist/react-formbuilder.js)
                      as directory
                        /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js doesn't exist
                      no extension
                        Field 'browser' doesn't contain a valid alias configuration
                        /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js doesn't exist
                      .js
                        Field 'browser' doesn't contain a valid alias configuration
                        /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js.js doesn't exist
                      .json
                        Field 'browser' doesn't contain a valid alias configuration
                        /Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js.json doesn't exist
[/Users/mavisou/Development/node_modules]
[/Users/mavisou/node_modules]
[/Users/node_modules]
[/node_modules]
[/Users/mavisou/Development/network-pulse/pages/node_modules]
[/Users/mavisou/Development/network-pulse/pages/add/node_modules]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder.js]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder.json]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index.js]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/index.json]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js.js]
[/Users/mavisou/Development/network-pulse/node_modules/react-formbuilder/dist/react-formbuilder.js.json]
 @ ./pages/add/add.jsx 13:24-52
 @ ./routes.jsx
 @ ./main.jsx

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build-client:js"
npm ERR! node v6.9.1
npm ERR! npm  v4.1.1
npm ERR! code ELIFECYCLE
npm ERR! network-pulse@2.0.0 build-client:js: `webpack --config webpack.config.js --optimize-minimize --display-error-details`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the network-pulse@2.0.0 build-client:js script 'webpack --config webpack.config.js --optimize-minimize --display-error-details'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the network-pulse package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --config webpack.config.js --optimize-minimize --display-error-details
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs network-pulse
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls network-pulse
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mavisou/Development/network-pulse/npm-debug.log
ERROR: "build-client:js" exited with 1.

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build:client"
npm ERR! node v6.9.1
npm ERR! npm  v4.1.1
npm ERR! code ELIFECYCLE
npm ERR! network-pulse@2.0.0 build:client: `run-s build-client:**`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the network-pulse@2.0.0 build:client script 'run-s build-client:**'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the network-pulse package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     run-s build-client:**
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs network-pulse
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls network-pulse
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mavisou/Development/network-pulse/npm-debug.log
ERROR: "build:client" exited with 1.

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v6.9.1
npm ERR! npm  v4.1.1
npm ERR! code ELIFECYCLE
npm ERR! network-pulse@2.0.0 build: `run-s bootstrap build:**`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the network-pulse@2.0.0 build script 'run-s bootstrap build:**'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the network-pulse package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     run-s bootstrap build:**
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs network-pulse
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls network-pulse
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mavisou/Development/network-pulse/npm-debug.log

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.9.1
npm ERR! npm  v4.1.1
npm ERR! code ELIFECYCLE
npm ERR! network-pulse@2.0.0 start: `npm run build && run-p server watch:**`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the network-pulse@2.0.0 start script 'npm run build && run-p server watch:**'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the network-pulse package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build && run-p server watch:**
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs network-pulse
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls network-pulse
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mavisou/Development/network-pulse/npm-debug.log
mavisou:~/Development/network-pulse[issue-309-form*]$ 
```

Adding https://webpack.js.org/configuration/resolve/#resolve-aliasfields to webpack.config.js should resolve this problem.

**Comment by Pomax** [comment]
pretty sure this was caused by the `dist` dir being in gitignore, which it shouldn't have been. v0.1.1 has been published to npm, please give that a try

**Comment by Pomax** [comment]
fixed with v0.1.1
